src/pudl/metadata/sources.py

```diff
@@ -753,7 +753,9 @@
             "source_format": "Parquet",
         },
         "working_partitions": {
-            "years": sorted(set(range(2019, 2024))),
+            "years": sorted(
```

Comment:
Any reason to not use `list(range(2021, 2024))`? Or just `[2021, 2022, 2023]`?

Reply:
I believe either of those would work! But I was just repeating the pattern for a range of years across the other sources in here.

I personally like the `range` because it makes sure you don't skip years. it feels a lil silly for three years but i think it is good practice. when i was messing around in a notebook and I was trying the 2019-2023 range i accedenitally skipped 2020.

Reply:
Yeah, I can see that - I think `list(range(...))` seems more concise / least surprising - `sorted(set(range(...)))` says "make this range, throw away the order, then re-order it" vs. "make this range, and turn it into a list."

Reply:
that makes a lo of sense! i changed this one. i am a lil tempted to just go change em all but i will refrain

Reply:
Maybe as a follow-up PR 😉